oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -415,9 +411,9 @@ static ExternalAccountCredentials fromJson(
     String clientSecret = (String) json.get("client_secret");
     String quotaProjectId = (String) json.get("quota_project_id");
     String userProject = (String) json.get("workforce_pool_user_project");
-    String universeDomain = (String) json.get("universe_domain");
     Map<String, Object> impersonationOptionsMap =
         (Map<String, Object>) json.get("service_account_impersonation");
+    String universeDomain = (String) json.get("universe_domain");
```

Comment:
nit: move this back under userProject to avoid change noise?

Reply:
yep, thanks!